packages/sales-api-service/src/services/transactions/__tests__/process-transaction-queue.spec.js

```diff
@@ -189,6 +187,8 @@ describe('transaction service', () => {
       mockRecord.permissions[0].isLicenceForYou = true
       AwsMock.DynamoDB.DocumentClient.__setResponse('get', { Item: mockRecord })
       await processQueue({ id: mockRecord.id })
+
+      // assert
```

Comment:
Is this necessary?

Reply:
Argh, I was showing something to Laila and obviously committed this by accident!